src/lib/stores.ts

```diff
@@ -1,12 +1,9 @@
 import { writable, type Writable } from 'svelte/store';
 import { ActionForms, defaultDotApi, type MsaInfo } from '$lib/storeTypes';
-
 export const storeConnected = writable(false);
 
 //All accounts
 export const storeValidAccounts = writable({});
```

Comment:
Should storeValidAccounts also be in accountsStore?

Reply:
Nice catch.  I've implemented this change.